static/js/trailTraffic.js

```diff
@@ -129,7 +129,13 @@ var ViewModel = function() {
     map.setCenter(redwoodCenter);
   });
 
-  var segmentLines = [];
+  self.infoTooltip = ko.observable("Trailways helps horseback riders and other " +
```

Comment:
The point being that this isn't really data, though.  It's part of the presentation.  There's nothing dynamic in here.  I feel you if this is part of the Udacity requirements, but in practice I wouldn't do this.